pkg/builder/openapi.go

```diff
@@ -83,22 +95,25 @@ func (o *openAPI) init(webServices []*restful.WebService) error {
 	if o.config.CommonResponses == nil {
 		o.config.CommonResponses = map[int]spec.Response{}
 	}
-	err := o.buildPaths(webServices)
-	if err != nil {
-		return err
-	}
+	return o
+}
+
+// getSwagger is called after the spec is built and returns the final spec.
+// NOTE: getSwagger also make changes to the final spec, as specified in the config.
+func (o *openAPI) getSwagger() (*spec.Swagger, error) {
```

Comment:
Consider the name "finalizeSwagger" to emphasize that the swagger is modified before being returned

Reply:
Done.
Sorry, can you lgtm again? I had to recommit.